.changeset/violet-dolls-arrive.md

```diff
@@ -0,0 +1,7 @@
+---
+'example-app': patch
+'@backstage/plugin-chrome-ux-report': patch
+'@backstage/plugin-chrome-ux-report-backend': patch
```

Comment:
No need to include a changeset, it'll just trigger another version bump after the initial release.